docs/tutorial.rst

```diff
@@ -186,7 +184,8 @@ of size 256.
 >>> data_stream = DataStream(mnist, iteration_scheme=SequentialScheme(
 ...     num_examples=mnist.num_examples, batch_size=256))
 
-As our algorithm we will use straightforward SGD with a fixed learning rate.
+Regarding the training algorithm, we will use straightforward SGD with a fixed
```

Comment:
Not sure I find this an improvement. Simpler is better, so I would go with "The training algorithm we will use is straightforward SGD with a fixed learning rate."